doc/en/components/notifications/banner.md

```diff
@@ -56,22 +116,34 @@ The `Banner` component allows templating of its content while still sticking as
 
 ### Changing the banner message
 
-Configuring the message displayed in the banner is easy - just change the content you are passing to the `igc-banner` tag. The text will show up in the specified banner area and the banner will use its default template when displaying it. Below, we will change the content of our sample banner to be a bit more descriptive:
+Configuring the message displayed in the banner is easy - just change the content you are passing to the <!-- WebComponents -->`igc-banner`<!-- end: WebComponents --><!-- React -->`IgrBanner`<!-- end: React --><!-- Blazor -->`IgbBanner`<!-- end: Blazor --> tag. The text will show up in the specified banner area and the banner will use its default template when displaying it. Below, we will change the content of our sample banner to be a bit more descriptive:
```

Comment:
I would suggest adding `BannerSelector` variable similar to the `GridSelector` one for each platform in the `docConfig.json`, so there would be not need for such hard to read conditions inside a text paragraph.